scout/server/blueprints/clinvar/templates/clinvar/multistep_add_variant.html

```diff
@@ -220,7 +220,7 @@ <h3 class="fs-subtitle">Represents a variant-level classification for a disease,
                   </div>
                 </div>
                 <br><br><br>
-                {{variant_data.var_form.clinsig_comment.label(class="fw-bold, text-dark")}} <span class="text-danger" data-bs-toggle='tooltip' title="Required only if you select 'drug response' or 'other' as clinical significance"><strong>?</strong></span>
+                {{variant_data.var_form.clinsig_comment.label(class="fw-bold, text-dark")}} <span class="text-danger" data-bs-toggle='tooltip' title="Optional, but highly encouraged. Free text describing the rationale for the classification."><strong>?</strong></span>
```

Comment:
Fixed according to their schema --> https://github.com/ncbi/clinvar/blob/408f3d0d44ac095e1f1f8737e7dafe751af7140b/submission_api_schema/submission_api_schema.json#L1370